pykafka/partitioners.py

```diff
@@ -39,6 +34,20 @@ def __call__(self, partitions, key=None):
                                   ' partitioner implementation')
 
 
+class RandomPartitioner(BasePartitioner):
```

Comment:
Since this isn't actually random anymore, I think a more accurate name would be better. Maybe `RoundRobinPartitioner`?

Also the docstring should be updated to indicate that this is not actually a random partition.